DCOS/templates/preprovision/extensions/preprovision-agent-windows/v1/preprovision-agent-windows-fluentd.ps1

```diff
@@ -0,0 +1,65 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
```

Comment:
If you moved the `preprovision-agent-windows.ps1` from [here](https://github.com/Microsoft/mesos-jenkins/tree/master/DCOS/fluentd-testing), please delete it from the previous location.